filament/backend/src/metal/MetalDriver.mm

```diff
@@ -1821,25 +1820,35 @@
     auto [fragment, vertex] = functions.getRasterFunctions();
 
     // Pipeline state
+    std::vector<NSUInteger> sampleCounts;
```

Comment:
do we really need a std::vector here? bindPipeline is on the critical path, I'd rather avoid heap allocations